chunky/src/java/se/llbit/chunky/chunk/ChunkData.java

```diff
@@ -0,0 +1,23 @@
+package se.llbit.chunky.chunk;
+
+import se.llbit.nbt.CompoundTag;
+
+import java.util.Collection;
+
+public interface ChunkData {
+  int getBlockAt(int x, int y, int z);
+
+  void setBlockAt(int x, int y, int z, int block);
+
+  boolean isBlockOnEdge(int x, int y, int z);
+
+  Collection<CompoundTag> getTileEntities();
+
+  Collection<CompoundTag> getEntities();
+
+  byte getBiomeAt(int x, int y, int z);
+
+  void setBiomeAt(int x, int y, int z, byte biome);
```

Comment:
IIRC biomes are not bytes anymore

Reply:
Chunky represented them as bytes previously, do you want me to move to `int[]` in this PR?

Reply:
Looked into the Biomes class… byte[] is fine :+1: